server/conf/cms/secrets.sample.py

```diff
@@ -3,33 +3,7 @@
 ########################
 
 SECRET_KEY = 'replacethiswithareallysecureandcomplexsecretkeystring'
-DEBUG = False
-
-ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost', '*']
-
-LDAP_ENABLED = False
-
-########################
-# DATABASE SETTINGS
-########################
-
-DATABASES = {
-   'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'PORT': '5432',
-        'NAME': 'taccsite',
-        'USER': 'postgresadmin',
-        'PASSWORD': 'taccforever',  # Change before live deployment.
-        'HOST': 'core_cms_postgres'
-    }
-}
```

Comment:
I thought the readme said to copy this stuff to correct file when setting up CMS. If this is deleted, then (1) will readme instructions be inaccurate, and (2) will dev still have enough direction to set up portal cms?

Reply:
This is defined by default in the Core-CMS repo, so I found it redundant to keep here.